autostart/qubes-app-menu.desktop

```diff
@@ -1,9 +1,9 @@
 [Desktop Entry]
 Type=Application
-Exec=systemctl --user start qubes-app-menu
+Exec=sh -c "systemctl --user import-environment XDG_CURRENT_DESKTOP; systemctl --user start qubes-app-menu"
```

Comment:
Can you add also `XDG_*` variables from https://specifications.freedesktop.org/basedir-spec/basedir-spec-latest.html#variables ? And maybe `XDG_SESSION_*` too?